FSW33-Team1/Challenge11
# Feature/be controller unit test

How to test on your PC :

1. Make sure you have installed all dependencies with `npm install` command
2. Execute all jest test with `npm run test` command
3. Use `npm run testWithCoverage` to get coverage test summary
